lib/tm4b/broadcast.rb

```diff
@@ -40,9 +40,7 @@ def split_method=(method)
       end
 
       def encoded_message
-        @message.
-          encode!("ISO-8859-1", :invalid => :replace, :undef => :replace, :replace => "").
-          encode!("UTF-8")
+        @message.encode!("UTF-8")
```

Comment:
Here, we were filtering out all non `ISO-8859-1` (latin alphabet) characters, then re-encoding the message in utf-8. Now we want to be able to send non-latin characters such as korean characters for Dior. So we do want to remove the latin alphabet filter.